virtual_workstation_setup.sh

```diff
@@ -1,18 +1,22 @@
 # Operations in this file should be idempotent-ish. At the very least, make
 # sure your code doesn't do duplicate work/clobber existing files on rerun
+set -o errexit
```

Comment:
The error that broke this script was transparently swallowed because we didn't set this before.